main.js

```diff
@@ -12,4 +12,133 @@ popUp.addEventListener('click', () => {
 
 closeMenu.addEventListener('click', () => {
   popUp.style.display = 'none';
-});
\ No newline at end of file
+});
+
+const modalObj = {
+  modal1: {
+    title: 'Tonic',
+    subt: ['CANOPY', 'Back End Dev', '2015'],
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+    img: 'img/Portfolio1-big.svg',
+    tech: ['html', 'css', 'javaScript'],
+    live: 'https://enekwestanley.github.io/Portifolio-page/',
+    source: 'https://github.com/Enekwestanley/Portifolio-page',
+  },
+  modal2: {
+    title: 'Multi-Post Stories',
+    subt: ['CANOPY', 'Back End Dev', '2015'],
+    desc: "Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged. It was popularised in the 1960s with the release of Letraset sheets containing Lorem Ipsum passages, and more recently with desktop publishing software like Aldus PageMaker including versions of Lorem Ipsum.",
+    img: 'img/Portfolio2-big.svg',
+    tech: ['html', 'css', 'javaScript'],
+    live: 'https://enekwestanley.github.io/Portifolio-page/',
+    source: 'https://github.com/Enekwestanley/Portifolio-page',
+  },
```

Comment:
- [ ] You have currently implemented an object which in turn contains your projects which are objects as well. A better way to do this would be to have an array of objects instead so you can easily loop over your projects 🗄️. The projects array would be an array of objects containing all your projects and necessary details about the projects. For the avoidance of doubt, each project would be an object in the projects array.

The projects array should look something like this:
```
const projects = [
  {
    name: "My project",
    description: "My project's description",
    image: "image.png",
    link: "live_link.com",
  }
]
```